Makefile

```diff
@@ -0,0 +1,38 @@
+.DEFAULT_GOAL := help
+
+# Parse Makefile and display the help
+help: ## Show help
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
A bit of magic to auto-generate usage docs for make targets:

![image](https://user-images.githubusercontent.com/1120468/207297647-4d052586-95e2-480a-a5d0-5be071c8a125.png)

Credits go to https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html